examples/tts/conf/fastpitch_align_44100_adapter.yaml

```diff
@@ -35,10 +35,9 @@ model:
   learn_alignment: true
   bin_loss_warmup_epochs: 100
   
-  n_speakers: 1
+  n_speakers: 1 # [TODO] removed after we change previous config
```

Comment:
Should we make this `??? ` so user knows they have to provide it?

Reply:
Yes, it can be `???`. But I think this argument can be placed into `speaker_encoder.lookup_module`.

Reply:
I see. But this parameter is not used if we don't use lookup module. So, do we still need `???`?

Reply:
The config is just an example, but it should be a complete example. If the example includes the lookup module, then n_speakers should be ??? because the adapter system does not make sense with n_speakers = 1. If there were a different example config without lookup module, then we could remove it.

Reply:
Ok, that is fine.